cmd/install/install.go

```diff
@@ -69,7 +71,8 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.ExcludeEtcdManifests, "exclude-etcd", false, "Leave out etcd manifests")
 	cmd.Flags().BoolVar(&opts.EnableOCPClusterMonitoring, "enable-ocp-cluster-monitoring", opts.EnableOCPClusterMonitoring, "Development-only option that will make your OCP cluster unsupported: If the cluster Prometheus should be configured to scrape metrics")
 	cmd.Flags().BoolVar(&opts.EnableCIDebugOutput, "enable-ci-debug-output", opts.EnableCIDebugOutput, "If extra CI debug output should be enabled")
-	cmd.Flags().StringVar(&opts.AWSCreds, "aws-creds", opts.AWSCreds, "Path to an AWS credentials file (AWS only)")
+	cmd.Flags().StringVar(&opts.AWSCreds, "aws-creds", opts.AWSCreds, "Path to an AWS credentials file")
```

Comment:
This is assuming every HC shares the same aws account for these resources, which is probably a valid assumption, though I think tangentially to this PR we need to clarify on our supported/desired deployment operator model.
We need set design invariants on how a single operator can/should/must support different tenants, providers and accounts.

Reply:
Yes, I think we arrived at:

`hypershift-operator` should support guest clusters with public endpoint access for any platform in any region.  It can only support guest clusters with private endpoint access on a single platform type in a single region.